auth-server-open-api-spec.yaml

```diff
@@ -837,8 +795,6 @@ components:
         access:
           type: array
           description: A description of the rights associated with this access token.
-          minItems: 1
-          maxItems: 1
```

Comment:
Is it possible to enforce that an `access`'s `types` can only appear once in the array?

Reply:
No it's not. I googled it and [it seems to still be impossible](https://stackoverflow.com/questions/67685894/openapi-3-0-how-to-define-uniqueness-of-array-elements-based-upon-a-key-proper).

Reply:
Possible alternatives:
- use a [tuple](https://json-schema.org/understanding-json-schema/reference/array.html#tuple-validation) such that each access array contains every `type` but allow `actions` to be empty
- some roundabout way of utilizing [`maxContains`](https://json-schema.org/understanding-json-schema/reference/array.html#mincontains-maxcontains)

Reply:
- https://github.com/interledger/open-payments/issues/171